### PR TITLE
Parameters adjustment

### PR DESCRIPTION
**Pull request by dfquaresma**
Update configuration file modifying and adding parameters and execution flows changing basic behaviors.

### CODE REVIEW

File: __main__.py [hunk 1]
```diff
@@ -60,11 +73,30 @@ def main():
         load_balancer.add_server(server)
         servers.append(server)
 
-    env.run(until=SIM_DURATION_SECONDS)
+    before_count = NUMBER_OF_SERVERS * [0]
+    before_time = NUMBER_OF_SERVERS * [0]
+    gc_count = list()  # how much gc executions was runned per second.
```
Comment (danielfireman) [inline __main__.py]: # Number of GC executions per second.

File: __main__.py [hunk 2]
```diff
@@ -60,11 +73,30 @@ def main():
...
+    gc_time = list()  # how much time was used gcing per second.
```
Comment (danielfireman) [inline __main__.py]: # Amount of time collecting garbage per second.

File: __main__.py (hunk 1 above)
Comment (dfquaresma) [reply __main__.py]: I'll fix it on the current branch that I'm work on. Thanks!

File: __main__.py (hunk 2 above)
Comment (dfquaresma) [reply __main__.py]: I'll fix it on the current branch that I'm work on. Thanks!

**Pull request merged**